src/datatrove/utils/word_tokenizers.py

```diff
@@ -8,6 +8,17 @@ def strip_strings(els: list[str]) -> list[str]:
     return [el.strip() for el in els if len(el.strip()) > 0]
 
 
+def simple_span_tokenize(text: str, sents: list[str]) -> list[tuple[int, int]]:
```

Comment:
let's maybe turn this function into a generator (remove the array and instead of appending, yield)